anvil/anvil.go

```diff
@@ -18,9 +18,12 @@ import (
 )
 
 type Config struct {
-	ChainId uint64
-	Port    uint64
-	Genesis []byte
+	ChainId        uint64
+	Port           uint64
+	Accounts       uint64
+	Mnemonic       string
+	DerivationPath string
```

Comment:
Instead of having the derivation/Mnemoic/Account be apart of the config, lets simply hardcode this?

I think we should be restrictive with config unless necessary